### ISSUE TITLE
ssl client endpoint string parser still enables SSLv3 unconditionally

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7686]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#7686|
|Type|defect|
|Created|2014-10-16 19:45:21Z|
|Branch|https://github.com/twisted/twisted/tree/no-ssl3-string-client-7686|

Like [#7684](https://github.com/twisted/twisted/issues/7684) but for clients, and even worse because there isn't even a "method" string parameter.

Attachments:

* [t7686.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/af0/af052f2ee76cd4e8a2340b8c02776ead3b9aaff3/3e719023057c32e7345ce24585f1263dad13984c.diff) (1572 bytes) - added by Alex on 2014-10-16 19:49:06Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7686 7686
type__defect defect
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_no_ssl3_string_client_7686 branches/no-ssl3-string-client-7686
branch_author__glyph glyph
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1413488721537336 1413488721537336
changetime__1474198049187875 1474198049187875
version__None None
owner__glyph glyph

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Alex set owner to @glyph|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_2"></a>|@glyph commented|
|-|-|

(In [43396]) Branching to no-ssl3-string-client-7686.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

patch committed & sent to buildbots.

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set status to `closed`|
|-|-|

(In [43398]) Merge no-ssl3-string-client-7686: disable SSLv3 in client string
endpoint parser

Author: Alex

Reviewer: glyph

Fixes: [#7686](https://github.com/twisted/twisted/issues/7686)

Use the default, correct behavior that Hynek has graciously bestowed
upon us.  We should be grateful for it.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_5"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
